apps/web/screens/Search/Search.tsx

```diff
@@ -255,93 +255,88 @@ const Search: Screen<CategoryProps> = ({
                   <>
                     <Filter
                       truncate
-                      selected={!filters.category && !filters.showNews}
+                      selected={!filters.category && !filters.type}
                       onClick={() => onRemoveFilters()}
-                      text={`${n(
-                        'allCategories',
-                        'Allir flokkar',
-                      )} (${grandTotalSearchResultCount})`}
+                      text={`${n('allCategories', 'Allir flokkar')} (${
+                        sidebarData.totalTagCount
+                      })`}
                       className={styles.allCategoriesLink}
                     />
                     <SidebarAccordion
                       id="sidebar_accordion_categories"
                       label={''}
                     >
                       <Stack space={[1, 1, 2]}>
-                        {sidebarCategories.map((c, index) => {
-                          const selected = c.key === filters.category
-                          const text = `${c.title} (${c.total})`
-
-                          if (c.key === 'uncategorized') {
-                            return null
-                          }
-
-                          return (
-                            <Filter
-                              key={index}
-                              selected={selected}
-                              onClick={() => onSelectCategory(c.key)}
-                              text={text}
-                            />
-                          )
-                        })}
+                        {Object.entries(sidebarData.tags).map(
+                          ([key, { title, total }]) => {
+                            const selected = key === filters.category
+                            const text = `${title} (${total})`
+
+                            if (key === 'uncategorized') {
+                              return null
+                            }
+
+                            return (
+                              <Filter
+                                key={key}
+                                selected={selected}
+                                onClick={() =>
+                                  onSelectSidebarTag('category', key)
+                                }
+                                text={text}
+                              />
+                            )
+                          },
+                        )}
                       </Stack>
                     </SidebarAccordion>
                   </>
                 )}
               </Box>
             </Sidebar>
-            {totalNews > 0 && (
```

Comment:
Do we want to show the other categories sidebar if it has no results?

Reply:
This is valid, will fix in another PR since this got auto merged